common/sgx_iohandler/trusted_iohandler.h

```diff
@@ -0,0 +1,48 @@
+/* Copyright 2019 Intel Corporation
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*     http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+
+#include <string>
+#include <stdint.h>
+
+// Returns iohandler id corresponding to iohandler name
```

Comment:
Please move these function comments to the `trusted_iohandler.cpp` file instead of the .h header file.

Reply:
Done